README.md

```diff
@@ -165,52 +166,67 @@ Confirm that the data has indeed been migrated by checking pgAdmin 4.
 
 - Export database snapshot: `pg_dump -h localhost -U USERNAME whiteclaw -f EXPORT-FILENAME.sql`
 - Wipe the existing database:
-   - Enter postgres interactively as the `postgres` user: `psql -U postgres -h localhost -W`
-   - Drop the database: `drop database whiteclaw;`
-   - Create the database: `create database whiteclaw;`
+  - Enter postgres interactively as the `postgres` user: `psql -U postgres -h localhost -W`
+  - Drop the database: `drop database whiteclaw;`
+  - Create the database: `create database whiteclaw;`
 - Restore the snapshot: `psql -h localhost -U USERNAME whiteclaw < EXPORT-FILENAME.sql`
 
+## Running Dockerized Flask App
+
+- `cd` into the root of this repo
+- Create a database dump and save it to `docker_postgres_dump.sql` using `pg_dump --create -h localhost -U <USERNAME> whiteclaw -f docker_postgres_dump.sql`
```

Comment:
Is this necessary every time one spins the Docker up? This essentially means that you need to have both a local, non-Docker copy of the database in order to use Docker, right? That would seem to get rid of some of the advantages of having implemented Docker. Do you think that's true? If so, how can we get around it?

Reply:
A couple thoughts

1. I haven't tested out running the ETL with docker, I wanted to get this out first to unblock prototyping the new infrastructure. Once the ETL is dockerized we can add a docker script to populate the ETL and have the database dump into a persistent docker volume.
2. I wonder if we could grant prod database access using aws access keys. I am on the fence about letting this happen bc scared of ppl messing up but could docker with prod, eliminating local DB requirements.